### PR TITLE
feat:added text-utilities component

### PR DESCRIPTION
**Pull request by abhishek17551**
Screenshot (138)
Screenshot (139)
Screenshot (140)
Screenshot (141)

### CODE REVIEW

**Comment by shlok30** [comment]
Looks good to merge

File: pages/components/text-utilities/text-utilities.html [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+                        <h6>Heading written as h6</h6>
+                    </div>
+                </div>
+
+                <div>
+                    <iframe
+                        src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=solarized+dark&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522list%2520flex-start%2520padding-m%2522%253E%250A%2520%2520%2520%2520%253Ch1%253EHeading%2520written%2520as%2520h1%253C%252Fh1%253E%250A%2520%2520%2520%2520%253Ch2%253EHeading%2520written%2520as%2520h2%253C%252Fh2%253E%250A%2520%2520%2520%2520%253Ch3%253EHeading%2520written%2520as%2520h3%253C%252Fh3%253E%250A%2520%2520%2520%2520%253Ch4%253EHeading%2520written%2520as%2520h4%253C%252Fh4%253E%250A%2520%2520%2520%2520%253Ch5%253EHeading%2520written%2520as%2520h5%253C%252Fh5%253E%250A%2520%2520%2520%2520%253Ch6%253EHeading%2520written%2520as%2520h6%253C%252Fh6%253E%250A%253C%252Fdiv%253E"
+                        style="width: 100%; height: 335px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (anupkrbid) [inline pages/components/text-utilities/text-utilities.html:66]: avoid inline styles

File: pages/components/text-utilities/text-utilities.html [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+                    style="width: 100%; height: 335px; border:0; transform: scale(1); overflow:hidden;"
```
Comment (anupkrbid) [inline pages/components/text-utilities/text-utilities.html:89]: avoid inline styles

**Review by anupkrbid — COMMENTED** [review]
Looks good.
Just added  one suggestion

File: pages/components/text-utilities/text-utilities.html (hunk 1 above)
Comment (abhishek17551) [reply pages/components/text-utilities/text-utilities.html:66]: its for the code snippet.

File: pages/components/text-utilities/text-utilities.html (hunk 2 above)
Comment (abhishek17551) [reply pages/components/text-utilities/text-utilities.html:89]: its for the code snippet.

**Review by shlok30 — APPROVED** [review]
Looks good to merge

**Pull request merged**